src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        return randomColor;
```

Comment:
Don't forget to include access modifiers for your classes and methods. It's best practice to specify whether they are public, protected, or private. In this case, 'getRandomBall' should probably be public.